### DISCUSSION TITLE
Being able to see the traffic log when using "panel only mode"

### DISCUSSION THREAD

**Discussion by eleliope** [dannagle/PacketSender#360] [Q&A] [answered]
Hi Dan,
When I use Packet Sender launched in "panel only mode", I don't have access to the traffic log (or I'm not finding it). It is not picked up even if I have the normal mode open in the background. 
Is there any way to access the traffic log when using the program in panel only mode?

**Comment by dannagle** [comment]
I think this is the setting you want...

![image](https://github.com/dannagle/PacketSender/assets/1140573/60ff6cd1-af95-495b-959b-b8346bccebdb)

It is under Settings -> Display and the bottom right.


And make sure you have Log Traffic enabled (on the main window)

![image](https://github.com/dannagle/PacketSender/assets/1140573/679ddbbd-d614-433f-b6c8-30fd621e3709)


This will launch the main Packet Sender window and then launch your starter panel. Packets should be captured in the logs with the main Packet Sender window running in the background.

**Reply by eleliope** [reply]
Hi!
Sorry for the delay in replying. I think I didn't explain very well the doubt I had. The configuration that you propose in your answer I knew it, that's how I have configured the program, and, in this way, I can see indeed the traffic log.
But what I want is to be able to use the panel without editing options (the one launched using the command line option --starterpanel from the command line); and to be able to access the traffic log as well.
![Captura de pantalla 2024-02-21 112119](https://github.com/dannagle/PacketSender/assets/155072869/7a8e03f7-0325-448d-aff6-70ebb018bd04)

The --starterpanel command works ok, I have no problems with that. I've made a .bat file that opens the command line, runs the starter panel command in maximized mode, and closes cmd:

@echo off
cd C:\Program Files PacketSender
START /MAX packetsender.exe --starterpanel
EXIT

And I put it in the Start Menu, so that it starts by default when I turn on the computer. It works fine, it starts with the management panel maximized, and with the EDITING/VIEWING option hidden, so that the button script can't be modified accidentally.  

It is when I use this panel (the one I open with the command line --starter panel, which has no edit option) that I get no record in the traffic log; even if I open too the main window of packet sender in the background (with or without the panel with edit/wiew option launched)

Is there any way that packets sent using this panel, (the one I open with the command line --starter panel), leave a traffic log somewhere?

Thank you!

**Reply by dannagle** [reply, marked as answer]
Let me make sure I understand.

You want:
- Auto-launch the starter panel
- Have that panel be read-only
- With traffic logging

Is this correct?

That is currently not supported. That is a middle ground that I did not consider. I can see the value. It'd be like having a read-only mode with an audit trail.

**Reply by eleliope** [reply]
Yes, that's exactly what I meant.
It is true that it could be an interesting middle ground. It would provide a simple way of sending signals to someone without technical training, who might get nervous if unexpected windows open up. And at the same time, it would be possible to check what has happened in case of an error. 
(But I can live without it for the time being)
Thank you, Dan!
